src/components/NavBar.css

```diff
@@ -0,0 +1,21 @@
+.navBar {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin: 20px 80px;
+}
+
+.navBar-title {
+  font-size: 46px;
+}
+
+.navBar-ul {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+  font-size: 20px;
+}
```

Comment:
- [ ] Please kindly add the vertical line in-between the `nav-items` as we have it in the given template. I'm sure one of the essence of this milestone is to learn how to reproduce a given design, hiring managers are also on a look out for this quality as well.

## Wireframe
![Uploading png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text.png…]()

Reply:
resolved! ✔️